doc/source/tutorials/calibration.rst

```diff
@@ -128,32 +119,19 @@ complex pulse sequence. Therefore with start with that:
     natives = platform.natives.single_qubit[0]
 
     # create pulse sequence and add pulses
-    sequence = PulseSequence(
-        [
-            (
-                qubit.drive,
-                Pulse(duration=2000, amplitude=0.01, envelope=Gaussian(rel_sigma=5)),
-            ),
-            (qubit.probe, Delay(duration=sequence.duration)),
-        ]
-    )
+    sequence = PulseSequence()
+    sequence.concatenate(natives.RX.create_sequence())
```

Comment:
Indeed, these weren't available when I wrote this and they would cause `doctest` to fail. Now they are, so I used them in some examples in 75f12e8416c579396fd677bc2f3a453c85996c35.